src/ape/contracts/base.py

```diff
@@ -463,6 +463,32 @@ def __len__(self):
         logs = self.provider.get_contract_logs(self.log_filter)
         return sum(1 for _ in logs)
 
+    def __call__(self, *args: Any, **kwargs: Any) -> "MockContractLog":
```

Comment:
Is the quotes required?
```suggestion
    def __call__(self, *args: Any, **kwargs: Any) -> MockContractLog:
```